.github/workflows/docs.yml

```
@@ -158,6 +176,23 @@ jobs:
           publish_dir: doc/_build/html/
           cname: docs.pyvista.org
 
+      - name: Remove non-multi version folder
+        run: rm -rf doc
+
+      - name: Deploy on main multiversion
+        if: github.event_name == 'push'
+        uses: pyansys/actions/doc-deploy-dev@main
+        with:
```

We are declaring the `CNAME` as an environment variable in the CI/CD pipelines. This way, it is only specified in a single place.

```suggestion
        with:
          cname: ${{ env.DOCUMENTATION_CNAME }}
```